### PR TITLE
Bug fix for app mgmt policy appliesTo response

### PR DESCRIPTION
**Pull request by madansr7**
Fixing a bug in the docs.

### CODE REVIEW

File: api-reference/beta/api/appmanagementpolicy-list-appliesto.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Do not supply a request body for this method.
 
 ## Response
 
-If successful, this method returns a `200 OK` response code and a collection of [appManagementPolicy](../resources/appManagementPolicy.md) objects in the response body.
+If successful, this method returns a `200 OK` response code and a collection of [application](../resources/application.md) and [servicePrincipal](../resources/serviceprincipal.md) objects in the response body.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/appmanagementpolicy-list-appliesto.md:59]:
```suggestion
If successful, this method returns a `200 OK` response code and a collection of [application](../resources/application.md) or [servicePrincipal](../resources/serviceprincipal.md) objects in the response body.
```

**Review by FaithOmbongi — APPROVED** [review]
Hi @madansr7 - Please also update the v1.0 file.

**Comment by madansr7** [comment]
@FaithOmbongi sorry, i trigged the re-review accidentally. No changes made since you last approved.

**Comment by FaithOmbongi** [comment]
Hi @madansr7 - Please address the merge conflict and update the v1.0 doc as well. Thanks.

**Review by FaithOmbongi — APPROVED** [review]

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**